test/benchmark_test.cc

```diff
@@ -218,6 +218,15 @@ void BM_non_template_args(benchmark::State& state, int, double) {
 }
 BENCHMARK_CAPTURE(BM_non_template_args, basic_test, 0, 0);
 
+void BM_complexity_args(benchmark::State &state, int n) {
```

Comment:
I don't think this is the right place for these tests.
Maybe it should go to `test/complexity_test.cc`.